src/components/ExperienceContainer/Experience.jsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState } from "react";
+import BrushIcon from "../../icons/BrushIcon";
+import "./ExperienceContainer.css";
+
+export default function Experience({ experience, saveExperience }) {
+  const [isEdit, setIsEdit] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+  const [currExperience, setcurrExperience] = useState(experience);
+  const [id, setTimeoutId] = useState(null);
+
+  function handleSaveFormEvent(e) {
+    e.preventDefault();
+    setIsEdit(false);
+    saveExperience(currExperience);
+  }
+
+  function fetchGeneratedDescription() {
+    setLoading(true);
+    clearTimeout(id);
+
+    setTimeoutId(
+      setTimeout(() => {
+        console.log("Generating description...");
+        setcurrExperience({
+          ...currExperience,
+          description: "Generated description",
```

Comment:
This is the mock data as the backend PR is not yet merged, right? Can you add a comment for this here